msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/nestedAppAuth/TestCase2688459.java

```diff
@@ -40,6 +42,8 @@
 
 // Nested App auth silent request
 // https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2688459
+@SupportedBrokers(brokers = BrokerHost.class)
+@LocalBrokerHostDebugUiTest
```

Comment:
Why are two similar annotations required?

Reply:
I think one of them is for the pipeline to identify that it needs to run with brokerHost.

Reply:
I'm not sure why the pipeline needs to identify it. The test infra today has capability to skip (ignore) tests has broker variant used is not in the list of SupportBrokers for the test.

Reply:
I am looking at the daily pipeline code and I see that there is a separate annotation being used to run brokerHost tests. That is probably the reaosn why we have 2 annotation for all tests that need to run on brokerHost.
https://identitydivision.visualstudio.com/Engineering/_apps/hub/ms.vss-build-web.ci-designer-hub?pipelineId=1490&branch=master&nonce=vCbn0InOkEbjZ%2BgVrNEk0Q%3D%3D

Reply:
There's a separate stage to run with local brokerhost, distAutoBroker is not always usable for tests that require running on brokerhost, I made the change a while ago but i think some tests would only automate in a separate stage, and would otherwise fail to run.